airflow/contrib/utils/kubernetes_utils.py

```diff
@@ -70,3 +79,23 @@ def __init__(self,
         self.name = name
         self.image = image
         self.command = command
+
+
+_bash_safe_replacements = dict((c, '\\' + c) for c in " !\"$&'()*,;<>?[\\]^`{|}")
+_bash_safe_replacements['\t'] = '\\t'
+_bash_safe_replacements['\n'] = '\\n'
+
+
+def bash_safe_escape(value):
```

Comment:
this is news to me, great catch!

Reply:
Depending on how we pass args, it doesn't matter. Since we are now ensuring that we are passing items as arrays this will no longer be necessary. So that's some wasted effort 👎